### PR TITLE
Update README to add test related target

### PR DESCRIPTION
**Pull request by pratikjagrut**
This PR adds `test-e2e` target in README. It is an important target and should have a place in README.

### CODE REVIEW

**Comment by pratikjagrut** [comment]
@jmrodri and @AlexNPavel What do you guys think of this PR?

File: memcached-operator/README.md [hunk 1]
```diff
@@ -67,4 +67,8 @@ To uninstall all that was performed in the above step run `make uninstall`.
 [kubectl_tool]:https://kubernetes.io/docs/tasks/tools/install-kubectl/
 [docker_tool]:https://docs.docker.com/install/
 [operator_sdk]:https://github.com/operator-framework/operator-sdk
-[operator_install]:https://github.com/operator-framework/operator-sdk/blob/master/doc/user/install-operator-sdk.md
\ No newline at end of file
+[operator_install]:https://github.com/operator-framework/operator-sdk/blob/master/doc/user/install-operator-sdk.md
+
+### Run Tests
+
+Run `make test-e2e` to run integration tests. This command will execute many times the same e2e tests. Its purposes is just to illustrate few options.
```
Comment (jmrodri) [inline memcached-operator/README.md]: I would rephrase the above sentence. But it matches what is currently in the Makefile. I will approve this and submit another PR to rephrase the text.
